repositories.gradle

```diff
@@ -0,0 +1,26 @@
+repositories {
+    maven {
+        name "ChickenBones Maven"
+        url "https://chickenbones.net/maven/"
+    }
+    maven {
+        name "IC2 Maven"
+        url "https://maven.ic2.player.to/"
+    }
+    ivy {
+        name "BuildCraft"
+        artifactPattern "https://www.mod-buildcraft.com/releases/BuildCraft/[revision]/[module]-[revision](-[classifier]).[ext]"
+    }
```

Comment:
this hurts when you have dependencies on the project, especially if some artifact taken from ivy has no group id (even if it is not used in ivy) gradle will fail to parse POM from jitpack, you sure we can't get rid of ivy here? 

If not - then you should probably describe how to properly depend on this project (e.g. dev will need to have `<insert ivy repo example` to be able to get all dependencies required by this mod)

Reply:
Buildcraft is not set as a dep anyway, so might as well remove it

Reply:
Actually he just left multiple dependency out. See https://github.com/GTNewHorizons/Galacticraft/blob/MC1.7/build.gradle#L88-L104

Reply:
Because all those dependencies are not used in the code. Checked it for every one. This speeds up the dev process in the end.